File: Java/opentraceability/src/main/java/opentraceability/models/events/EventBusinessTransaction.java
package opentraceability.models.events;

import opentraceability.utility.attributes.OpenTraceabilityAttribute;
import opentraceability.utility.attributes.OpenTraceabilityJsonAttribute;

import java.net.URI;

public class EventBusinessTransaction {

    @OpenTraceabilityJsonAttribute(name="type")
    @OpenTraceabilityAttribute(ns = "", name = "@type")
    public URI type;

    @OpenTraceabilityJsonAttribute(name="bizTransaction")
    @OpenTraceabilityAttribute(ns = "", name = "text()")
    public String value;
}

File: Java/OpenTraceability/src/main/kotlin/opentraceability/models/events/eventbusinesstransaction.kt
package models.events

import utility.attributes.*
import java.net.URI

class EventBusinessTransaction {

    @OpenTraceabilityJsonAttribute("type")
    @OpenTraceabilityAttribute("", "@type")
    var Type: URI? = null

    @OpenTraceabilityJsonAttribute("bizTransaction")
    @OpenTraceabilityAttribute("", "text()")
    var Value: String? = null

}
